### PR TITLE
Editorial review of updated PGD4PK content

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/postgres_distributed_for_kubernetes/1/before_you_start.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+EDB Postgres Distributed for Kubernetes requires a Kubernetes version supported by the community. See
+[Supported releases](https://www.enterprisedb.com/resources/platform-compatibility#pgdk8s) for details.
+
+## PGD terminology
+
+For more information, see
+[Terminology](/pgd/latest/terminology/) in the PGD documentation.
+
+[Node](/pgd/latest/terminology/#node)
```
Comment (gvasquezvargas) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/before_you_start.mdx:85]:
```suggestion
[Data node](/pgd/latest/terminology/#node)
```
Discussed this change with @jsilvela today, so it is less ambiguous.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/before_you_start.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  storage.
+
+[Service](https://kubernetes.io/docs/concepts/services-networking/service/)
+: A *service* is an abstraction that exposes as a network service an
+  application that runs on a group of pods and standardizes important features,
+  such as service discovery across applications, load balancing, and failover.
+
+[Secret](https://kubernetes.io/docs/concepts/configuration/secret/)
+: A *secret* is an object that's designed to store small amounts of sensitive
+  data, such as passwords, access keys, or tokens, and use them in pods.
```
Comment (gvasquezvargas) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/before_you_start.mdx:27]:
```suggestion
  data, such as passwords, access keys, or tokens, and uses them in pods.
```
I am assuming the subject here is the secret?

**Comment by gvasquezvargas** [comment]
@ebgitelman not sure, but I think this PR should be committing to `docs/pg4k-pgd/v1-import` instead of to `develop`? 
cc: @josh-heyer for clarification, cause I will need to do a similar procedure a couple of times next week.

**Comment by josh-heyer** [comment]
That is correct, @gvasquezvargas - I've corrected this to point to the intended base branch.

**Comment by josh-heyer** [comment]
⚠Rebased against latest import from PG4K repo - now includes new TDE topic

**Comment by josh-heyer** [comment]
⚠Rebased against latest v1-import (complete set of topics)

File: product_docs/docs/postgres_distributed_for_kubernetes/1/before_you_start.mdx (hunk 2 above)
Comment (josh-heyer) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/before_you_start.mdx:27]: Neither of these is quite accurate; a secret doesn't itself use anything, rather it is available for use. And while secrets could be used for data exchange, there's no particular requirement that any pod be able to update them. I'm gonna suggest 

```suggestion
  data, such as passwords, access keys, or tokens, for use within pods.
```

...as I think that's a definition that's accurate enough for the purposes of configuring PG4K-PGD (which is our goal here) - we link to the full docs for folks who want a more rigorous definition!

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**